test/path_template.ts

```diff
@@ -169,8 +167,9 @@ describe('PathTemplate', () => {
       '/buckets/helloazAZ09-.~_what': 'buckets/helloazAZ09-.~_what',
     };
 
-    _.forEach(tests, (want, template) => {
-      it('should render template ' + template + ' ok', () => {
+    Object.keys(tests).forEach(want => {
```

Comment:
nit: this would be more readable:

```ts
    for (let want in tests) {
      const template = tests[want];
      it(`should render template ${template} ok`, () => {
        const t = new PathTemplate(template);
        expect(t.inspect()).to.eql(want);
      });
    }
```